cmd/internal/flowmode/cmd_run.go

```diff
@@ -217,7 +233,7 @@ func (fr *flowRun) Run(configFile string) error {
 		// will take precedence over anything else mapped in uiPrefix.
 		ui.RegisterRoutes(fr.uiPrefix, r)
 
-		srv := &http.Server{Handler: r}
+		srv := &http.Server{Handler: h2c.NewHandler(r, &http2.Server{})}
```

Comment:
Are we okay with enabling h2c for all routes here?

Reply:
I'd say so 👍